### PR TITLE
Added Universities-Module in HR Module

### PR DESCRIPTION
**Pull request by maniSHarma7575**
@rathorevaibhav Please review the same and provide further feedback.

**What I did yet:**

- Added Universities Listing table.

- CRUD operation for universities and universities contacts.

- Worked on the design feedbacks.

### CODE REVIEW

File: resources/views/hr/universities/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        </div>
+        <div class="card-footer">
+            <a href="#" onClick="addNewContactForm({{old('email')==''?false:true}})">Add More Contact</a>
+        </div>
+    </div> 
+</div>
+@endsection
+
+@section('inline_js')
+function addNewContactForm(opened){
```
Comment (rathorevaibhav) [inline resources/views/hr/universities/edit.blade.php:121]: external js. use app.js

File: resources/views/hr/universities/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        <table class="table table-striped table-bordered" id="universities_table">
+            <tr>
+                <th>University</th>
+                <th>Contact Details</th>
+                <th>Action</th>
+            </tr>
+            @foreach($universities as $university)
+            <tr>
+            <td>
+            <h5 style="font-weight:bold">{{$university->name}}</h5>
```
Comment (rathorevaibhav) [inline resources/views/hr/universities/index.blade.php:42]:
```suggestion
            <h5 class='font-weight-bold'>{{$university->name}}</h5>
```

File: resources/views/hr/universities/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                @foreach($university->universityContacts as $contact)
+                <tr>
+                    <td>
+                    {{$contact->name}}
+                    </td>
+                    <td>
+                    {{$contact->email}}
+                    </td>
+                    <td>
+                    {{$contact->designation}}
```
Comment (rathorevaibhav) [inline resources/views/hr/universities/index.blade.php:57]:
```suggestion
                    {{$contact->designation ?? '-'}}
```

File: routes/web.php [hunk 4]
```diff
@@ -105,6 +105,26 @@
...
+            ->names([
+                'index' => 'universities',
+                'store' => 'universities.store',
+                'update'=>'universities.update',
+                'edit'=>'universities.edit',
+                'create'=>'universities.create',
+                'destroy'=>'universities.destroy'
+        ]);
+
+        Route::resource('universities.contacts', 'Universities\UniversityContactController')
```
Comment (rathorevaibhav) [inline routes/web.php:120]:
```suggestion
        Route::resource('universities/contacts', 'Universities\UniversityContactController')
```

File: routes/web.php [hunk 5]
```diff
@@ -105,6 +105,26 @@
...
+            ->only(['store','update','destroy'])
+            ->names([
+                'store'=>'universities.contacts.store',
+                'update'=>'universities.contacts.update',
+                'destroy'=>'universities.contacts.destroy'
+        ]);
+        Route::get('universities-reports', 'Universities\ReportController@index')->name('universities.reports');
```
Comment (rathorevaibhav) [inline routes/web.php:127]:
```suggestion
        Route::get('universities/reports', 'Universities\ReportController@index')->name('universities.reports');
```

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]

**Comment by maniSHarma7575** [comment]
Requested changes are done. @rathorevaibhav  sir, please review the same.

**Pull request closed without merging**